src/Traits/Categories/ButtonHelpers.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace NotificationChannels\OneSignal\Traits\Categories;
+
+use NotificationChannels\OneSignal\OneSignalButton;
+use NotificationChannels\OneSignal\OneSignalWebButton;
+
+trait ButtonHelpers
+{
+    /**
+     * Add a web button to the message.
+     *
+     * @param OneSignalWebButton $button
+     *
+     * @return $this
+     */
+    public function webButton(OneSignalWebButton $button)
+    {
+        return $this->setParameter('web_buttons', [$button->toArray()]);
+    }
+
+    /**
+     * Adds more than one web button to the message.
+     *
+     * @param array[OnSignalWebButton] $buttons
+     *
+     * @return $this
+     */
+    public function webButtons(array $buttons)
+    {
+        return $this->setParameter('web_buttons', collect($buttons)->map(function ($button) {
+            return $button->toArray();
+        }));
+    }
+
+    /**
+     * Add a native button to the message.
+     *
+     * @param OneSignalButton $button
+     *
+     * @return $this
+     */
+    public function button(OneSignalButton $button)
```

Comment:
do you think maybe this should be prefixed. For example - take the notifications conversation we had:

`incrementBadgeCount()` sound like it will do it everywhere.

`incrementIosBadgeCount()` sound like it will do it on iOS.

`webButtons` sounds like it is for the web (which is it)

`buttons` sounds like it will do it everywhere - web + native

Reply:
Yeah i agree with you, but i think this is something that should clarify first, since they named it "web buttons" and "buttons" 
https://documentation.onesignal.com/reference#section-action-buttons

Reply:
uh! yea - thats tough because I think matching their docs is good!